### DISCUSSION TITLE
如何减小打包体积？现在的太大了

### DISCUSSION THREAD

**Discussion by yu-sssss** [antvis/G6#3445] [Q&A] [answered]
现在的太大了，压缩后也有400多KB，我改成按需引入之后也才只减少了不到100KB。就只用了dagre，三个插件，几个自定义节点。
`import Graph from '@antv/g6-pc/es/graph/graph';`

**Comment by hustcc** [comment, marked as answer]
包大小，我们在 5.0 中考虑进去，可以关注：https://www.yuque.com/antv/g6/vdecqwim0inyga8a 6 月活发布 5.0 正式版本。
